docs/howtos/antora.yml

```diff
@@ -1,12 +1,12 @@
 name: axon_framework_ht
-title: Axon Framework
+title: Axon Framework Fundamentals
 version: true
 prerelease: true
 
 asciidoc:
   attributes:
-    component_description: The Axon Framework how-to guides
-    type: howto
+    component_description: A tutorial covering Axon Framework's fundamental components and features.
+    type: tutorial
```

Comment:
From the content so far, I come to two conclusions.
1) This is not a how-to as it is not problem-solving oriented. It's a tutorial, as it shows some possible usage of a given feature
2) It's too short for a stand-alone tutorial. Thus, it's better to have it as part of a larger tutorial.

So I propose to go with "Axon Framework Fundamentals" where we explain the fundament features of the framework.